mvc/src/test/java/webmvc/org/springframework/web/servlet/mvc/tobe/AnnotationHandlerMappingAdapterTest.java

```diff
@@ -0,0 +1,52 @@
+package webmvc.org.springframework.web.servlet.mvc.tobe;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import samples.TestController;
+import webmvc.org.springframework.web.servlet.ModelAndView;
+
+@SuppressWarnings("NonAsciiCharacters")
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class AnnotationHandlerMappingAdapterTest {
+
+    @Test
+    void supports_메서드는_HandlerExecution_타입이면_true_반환() throws NoSuchMethodException {
+        AnnotationHandlerMappingAdapter annotationHandlerMappingAdapter = new AnnotationHandlerMappingAdapter();
+        HandlerExecution handlerExecution = new HandlerExecution(new TestController(),
+                TestController.class.getMethod("findUserId", HttpServletRequest.class, HttpServletResponse.class));
```

Comment:
사소하지만

```suggestion
        HandlerExecution handlerExecution = new HandlerExecution(
                new TestController(),
                TestController.class.getMethod("findUserId", HttpServletRequest.class, HttpServletResponse.class)
        );
```

전 개인적으로 이렇게 개행을 하는 것이 좀 더 가독성이 좋아보이는데 어떻게 생각하시는지 궁금합니다!

Reply:
좋네요 ㅎㅎ 이런 소소한 부분 꼼꼼하게 봐주셔서 너무 감사합니다 !